### ISSUE TITLE
Remove duplicate spaces when returning title

### ISSUE THREAD

**Issue by jakob1379** [metebalci/pdftitle#9]
I found pdftitle which have proven to be a true gem! From time to time it to returns duplicate spaces in the title, which can easily be circumvented in two ways

1. using regular expression, just before returning `title` in `get_title_from_io` just add a 
```
titlte = re.sub(' +' , ' ', title)
``` 

2. using string manipulation
```
title = ' '.join(title.split())
```
Preferably a check with `'  ' in title` should be performed to check whether the correction should be executed or not.

**Comment by metebalci** [comment]
Would you like to send a patch ? or I can fix it. I think there is no need for a check and I would prefer the second method with split and join.

**Comment by jakob1379** [comment]
I just made a pull request :)

**Comment by metebalci** [comment]
Thanks. Merged and released with 0.4.
